src/game/client/components/menus_browser.cpp

```diff
@@ -1600,10 +1587,32 @@ void CMenus::RenderServerbrowserFriends(CUIRect View)
 
 			if(m_avFriends[FriendType].empty())
 			{
-				CUIRect Label;
-				List.HSplitTop(12.0f, &Label, &List);
-				s_ScrollRegion.AddRect(Label);
-				Ui()->DoLabel(&Label, Localize("None"), Label.h * CUi::ms_FontmodHeight, TEXTALIGN_ML);
+				const char *pText;
+				switch(FriendType)
+				{
+				case FRIEND_PLAYER_ON:
+					pText = Localize("Add friends by entering their name below or by clicking their name in the player list.");
+					break;
+				case FRIEND_CLAN_ON:
+					pText = Localize("Add clanmates by entering their clan below and leaving the name blank.");
+					break;
+				case FRIEND_OFF:
+					pText = Localize("Offline friends and clanmates will appear here.");
+					break;
+				default:
+					dbg_assert(false, "Invalid friend state");
+					dbg_break();
+				}
+				// Render empty description
+				const STextBoundingBox BoundingBox = TextRender()->TextBoundingBox(FontSize, pText, -1, List.w - 5.0f);
+				CTextCursor Cursor;
+				TextRender()->SetCursor(&Cursor, List.x, List.y + 2.0f, FontSize, TEXTFLAG_RENDER);
+				Cursor.m_LineWidth = List.w - 5.0f;
+				TextRender()->TextEx(&Cursor, pText, -1);
+				// Add text to List
+				CUIRect EmptyDescription;
+				List.HSplitTop(BoundingBox.m_H + 4.0f, &EmptyDescription, &List);
+				s_ScrollRegion.AddRect(EmptyDescription);
```

Comment:
```suggestion
				const float DescriptionMargin = 2.0f;
				const STextBoundingBox BoundingBox = TextRender()->TextBoundingBox(FontSize, pText, -1, List.w - 2 * DescriptionMargin);
				CUIRect EmptyDescription;
				List.HSplitTop(BoundingBox.m_H + 2 * DescriptionMargin, &EmptyDescription, &List);
				s_ScrollRegion.AddRect(EmptyDescription);
				EmptyDescription.Margin(DescriptionMargin, &EmptyDescription);
				SLabelProperties DescriptionProps;
				DescriptionProps.m_MaxWidth = EmptyDescription.w;
				Ui()->DoLabel(&EmptyDescription, pText, FontSize, TEXTALIGN_ML, DescriptionProps);
```

- Consistent margin on all sides.
- Prefer using `DoLabel` for UI instead of the textrender directly.
- Reserve the space in the scroll region before drawing the text for consistent usage of `CScrollRegion`.